### DISCUSSION TITLE
Can not install the mysql to my pc.

### DISCUSSION THREAD

**Discussion by Habibsultani** [community/community#75358] [Programming Help]
### Body

I using the 64.bit windows 11. Actually I have installed the mysql installer. but still is not installing the mysql.
![mysql ](https://github.com/community/community/assets/94174287/9c241215-d8bb-4634-8839-deacd26d0631)

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by elderguardian** [comment]
It is in offline mode, I guess you should  connect to the internet

**Reply by Habibsultani** [reply]
My PC is connected to the internet. The problem is not with the internet. I guess.

**Reply by elderguardian** [reply]
Well, you could check proxy and firewall settings. If there is nothing you could just install it manually

**Reply by Habibsultani** [reply]
Thank you I really appreciate it.
The problem was with my Laragon which I did not start the server database and other's option before running the mysql -u root commond.

![answer to mysql](https://github.com/community/community/assets/94174287/afc3a4a1-7774-432f-84ea-aaca7c5daf73)
